### PR TITLE
adds substantial completion date tooltip

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/19259

based off pr  https://github.com/cityofaustin/atd-moped/pull/1453

I believe the placement of this definition is centralized since we can import this helper object into the project view, but please let me know if there would be a better place to store it!

## Testing
**URL to test:** 

https://deploy-preview-1454--atd-moped-main.netlify.app/

**Steps to test:**
- in the projects list view, hover over 'Substantial completion date'
- note that a tooltip appears reading 'The earliest confirmed start date of a project phase that has a simple phase name of Complete (Complete or Post-Construction)'

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Product manager checked [DB view dependencies](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)
- [x] Product manager added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable
   - Added to sort by headers test

### CODE REVIEW

File: moped-editor/src/views/projects/projectsListView/helpers.js
Comment (mddilley) [inline moped-editor/src/views/projects/projectsListView/helpers.js:318]: @tillyw Could you update the description to say `Post-construction` so it matches the capitalization in the [data dictionary](https://mobility.austin.gov/moped/dev/lookups#moped-phases)? I made a typo and then copy-pasta'd into every issue involving this date? Just fixed those

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/helpers.js]: done!

**Review by chiaberry — APPROVED** [review]
✅  and I copied this text for the tooltip in my branch im working on

File: moped-editor/src/views/projects/projectsListView/helpers.js
Comment (mddilley) [inline moped-editor/src/views/projects/projectsListView/helpers.js:318]: 🙌 @chiaberry @tillyw Could y'all coordinate for someone to make this description into a variable to hold this string so we can reference it in this PR and in https://github.com/cityofaustin/atd-moped/pull/1456? We may end up adding this in the Data Dictionary too so it could help us update the definition if we ever need to in the future.

I looked in the codebase, and maybe we could add a new file in `src/constants` to hold tooltip text. What do y'all think?

Comment (chiaberry) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: I'll add this to my branch, trying to decide what to name the file, `projects.js` sound okay?

Comment (chiaberry) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: I did it here: https://github.com/cityofaustin/atd-moped/pull/1456/commits/a60747a304869b88970d5fea2db0458c69820cba

Comment (mddilley) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: thanks, @chiaberry!

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: @chiaberry sorry i merged your entire branch into this one when i should have just merged this one commit! i'll just make sure to merge this one last out of the substantial completion date PRs

Comment (mddilley) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: @tillyw sounds good! i'll hold off on approving this one since there are pending changes in Chia's PR. Thanks for planning ahead on the merge order since I brought Chia's issue into the mix late in the sprint. 🙏

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: @mddilley all is in order! this one is ready to merge once i get your approval :)

Comment (mddilley) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: @tillyw awesome - thanks for getting this one caught up to main branch! 🙏

Could you update this line to use the variable that Chia created in https://github.com/cityofaustin/atd-moped/pull/1456/files though? I'll approve asap after that. Thank you!!

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/helpers.js:318]: i'm sorry i did not think before i rerequested review on this >.< will fix

Comment (tillyw) [reply moped-editor/src/views/projects/projectsListView/helpers.js]: @mddilley imported!

**Review by mddilley — APPROVED** [review]
Looks ✨!! Thank you! 🚢

**Pull request merged**
